toolkit/tools/roast/roast.go

```diff
@@ -53,279 +36,26 @@ var (
 	imageTag = app.Flag("image-tag", "Tag (text) appended to the image name. Empty by default.").String()
 )
 
+func populateRoastConfig() *roast.Config {
+	return &roast.Config{
+		InputDir:       *inputDir,
+		OutputDir:      *outputDir,
+		ConfigFile:     *configFile,
+		TmpDir:         *tmpDir,
+		ReleaseVersion: *releaseVersion,
+		Workers:        *workers,
+		ImageTag:       *imageTag,
+	}
+}
+
 func main() {
```

Comment:
Please let's keep `main` as the first function after the arguments and consts list - I wouldn't want to change that "convention".